src/streamlink/plugins/mycujoo_tv.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import re
+
+from streamlink.compat import urlparse
+from streamlink.plugin import Plugin
+from streamlink.stream import HLSStream
+
+log = logging.getLogger(__name__)
+
+class MycujooTv(Plugin):
+    """
+    Support for live and archived transmission of the matches and channels on mycujoo.tv
+    """
+    url_re = re.compile(r'https?://mycujoo\.tv/video/.')
+    streams_re = re.compile(r'"filename"\:.*?"([^"]*?)"')
```

Comment:
this will **not work correctly**,
there are more than one filename.

If you want to make it work correctly,


1. you will have to parse `window._d` from the html source; `r'window\._d\s*=\s*({[^;]+});'` this will need `parse_json` and `validate` schema
2. you will have to parse the used URL for the highlight ID, video ID ...
3. use the found ID for the found json data, if there is no ID use the event / video / live data
4. select the right `filename` for vods or `['channel']['url_hls']` and `?start=` with the `startlive` time from event / live